libs/community/langchain_community/llms/vllm.py

```diff
@@ -5,6 +5,7 @@
 from langchain_core.outputs import Generation, LLMResult
 from langchain_core.utils import pre_init
 from pydantic import Field
+from vllm import SamplingParams
```

Comment:
can't improt from global namespace since vllm shouldn't be assumed to be available on user machine